docs/cave/goes-16-17-satellite.md

```diff
@@ -1,47 +1,53 @@
-# GOES 16/17
+# GOES East/West Satellite Products
 
-The ***goesr*** EDEX decoder supports the ingest of GOES products coming over NOAAPort and Unidata's IDD. These include [**single channel imagery**](#individual-channels), [**derived products**](#derived-products) (Level 2b netCDF files), gridded [**Geostationary Lightning Mapper**](#geostationary-lightning-mapper-glm) (GLM) products (produced by Eric Bruning at Texas Tech), [**CIRA created RGB**](#cira-geocolor) specific products, and [**vertical temperature/moisture profiles**](#vertical-temperature-and-moisture-profile). Using derived parameters, additional [**RGB**](#rgb-composites) and [**channel difference**](#channel-differences) products can be loaded. The ***dmw*** EDEX decoder supports the ingest of GOES [**derived motion winds**](#derived-motion-winds).
+The ***goesr*** EDEX plugin decoder supports the ingest of GOES products coming over NOAAPort and Unidata's IDD. These products include:
+- [single channel imagery](#individual-channels)
+- [derived products](#derived-products) (Level 2b netCDF files)
+- [CIRA created RGB](#cira-geocolor) specific products
+- [vertical temperature/moisture profiles](#vertical-temperature-and-moisture-profile). 
+- Using derived parameters, additional [RGB](#rgb-composites) and [channel difference](#channel-differences) products can be loaded.
+
+The ***dmw*** EDEX plugin decoder supports the ingest of GOES:
+  - [derived motion winds](#derived-motion-winds)
+  - [Geostationary Lightning Mapper](#geostationary-lightning-mapper-glm) (GLM) gridded products
+  - GLM Parallax corrected Flash Points
 
 ![](../images/GOESEW.png)
 
-GOES East and West products are accessible in the **Satellite** menu.  The menu is broken into sections starting with common CONUS GOES East/West Combo products. There are submenus for each of the separate geospatial sectors:
+GOES East and West products are accessible from the **Satellite** menu.  The menu is broken into sections starting with common CONUS GOES East/West Combo products. There are submenus for each of the separate geospatial sectors:
 
-* East Full Disk
 * East CONUS
+* East Full Disk
 * East Mesoscale Sectors (x2)
-* West Full Disk
 * West CONUS
+* West Full Disk
 * West Mesoscale Sectors (x2)
-* Hawaii
-* Alaska
-* Puerto Rico
+* GOES OCONUS
+  * Alaska
+  * Puerto Rico
   
-Each sector submenu has products for individual channels and vertical profiles, as well as submenus for derived products, channel differences, RGB Composites, GLM data, and derived motion winds. GLM data can also be found with its own submenu option a little lower down the menu and under the **Surface** menu.
-
-!!! warning "The RGB products are not available on MacOS or in a Virtual Machine running CAVE."
```

Comment:
Are you sure we want to remove this warning?  It's still applicable and seems like it might be nice to still have here?

Reply:
We had it in multiple places, so I kept the warning down in the actual RGB section where there were a few more details.